### ISSUE TITLE
Missing ground truth data for validation (./Valid/Validation/ground_truth)

### ISSUE THREAD

**Issue by Sevvvvvvvvv** [RiverNA/HookFormer#2]
Thank you for sharing this great repository! I am trying to run the validation phase of the model, but I noticed that the code requires a directory ./Valid/Validation/ground_truth which is not provided in the repository. How can I generate or obtain this data?

**Comment by RiverNA** [comment]
Hi @Sevvvvvvvvv, the validation set is split from the original training set of [CaFFe](https://github.com/Nora-Go/Calving_Fronts_and_Where_to_Find_Them) (please see its **Data Preparation**). CaFFe does not have a fixed validation set, you can also generate a new randomized validation-training-split.

**Comment by Sevvvvvvvvv** [comment]
> Hi [@Sevvvvvvvvv](https://github.com/Sevvvvvvvvv), the validation set is split from the original training set of [CaFFe](https://github.com/Nora-Go/Calving_Fronts_and_Where_to_Find_Them) (please see its **Data Preparation**). CaFFe does not have a fixed validation set, you can also generate a new randomized validation-training-split.

I am training the model on an RTX 4090 (24GB), but after 6 days, it has only reached the 53rd epoch. Is this expected? The paper does not mention training speed. Any insights on the expected training time?

**Comment by RiverNA** [comment]
Yes, we trained the model on an A100 (80GB), so this could be an issue for GPUs with lower memory. You might consider using multi-GPU training with [torch.nn.parallel.DistributedDataParallel](https://pytorch.org/tutorials/beginner/ddp_series_multigpu.html), accordingly you need to modify some codes to make it work properly.
